src/main/java/kr/hhplus/be/server/infrastructure/persistence/keyword/PopularKeywordRepositoryImpl.java

```diff
@@ -0,0 +1,48 @@
+package kr.hhplus.be.server.infrastructure.persistence.keyword;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Repository;
+
+import jakarta.transaction.Transactional;
+import kr.hhplus.be.server.domain.keyword.PopularKeyword;
+import kr.hhplus.be.server.domain.keyword.PopularKeywordRepository;
+
+@Repository
+public class PopularKeywordRepositoryImpl implements PopularKeywordRepository{
+    
+    private final JpaPopularKeywordRepository jpaRepository;
+
+    public PopularKeywordRepositoryImpl(JpaPopularKeywordRepository jpaRepository) {
+        this.jpaRepository = jpaRepository;
+    }
+
+    @Override
+    @Transactional
+    public void increaseCount(String keyword) {
+        PopularKeywordEntity entity = jpaRepository.findByKeyword(keyword)
```

Comment:
```suggestion
@Transactional
public void increaseCount(String keyword) {
    PopularKeywordEntity entity = jpaRepository.findByKeyword(keyword)
        .orElseGet(() -> new PopularKeywordEntity(keyword));

    entity.increaseCount();
    
    if (entity.getId() == null) {
        jpaRepository.save(entity); // 신규일 때만 save 필요
    }
}

```

기존 엔티티는 영속성 컨텍스트에 이미 존재하기에

변경만 하면 flush() 또는 트랜잭션 종료 시 자동 반영될것으로 예상되네요!